game/entities/sdFactionTools.js

```diff
@@ -206,6 +624,243 @@ class sdFactionTools extends sdEntity
 					setInterval( logic, 1000 );
```

Comment:
Intervals needs to be cleared with clearInterval - otherwise these will cause memory leaks overtime. If that is done in other places - those should be updated as well. @Booraz149 

Also, I do hope we don't have same code copied in more than 1 place in whole project.